composer.json

```diff
@@ -37,7 +37,7 @@
         "php": ">= 7.2",
         "illuminate/contracts": "^6.0|^7.0|^8.0",
         "illuminate/support": "^6.0|^7.0|^8.0",
-        "webonyx/graphql-php": "^14.0",
+        "webonyx/graphql-php": "^14.6.4",
```

Comment:
the reason why I upped this was because of a bug in the queryplan that fixed in this release